src/main/interface/cli.c

```diff
@@ -4504,6 +4522,7 @@ const clicmd_t cmdTable[] = {
     CLI_COMMAND_DEF("msc", "switch into msc mode", NULL, cliMsc),
 #endif
     CLI_COMMAND_DEF("name", "name of craft", NULL, cliName),
+    CLI_COMMAND_DEF("display_name", "display name of craft", NULL, cliDisplayName),
```

Comment:
We prefer commands to be added in alphabetical order. That way the `help` output is more useful.

Reply:
Little correction: We **require** commands to be in alphabetical order - if they aren't then tab completion in the CLI does not work properly.